app/views/components/_subscribe.html.erb

```diff
@@ -1,3 +1,3 @@
 <%= tag.div class: "app-c-subscribe" do %>
-  <%= link_to label, url, title: title, class: "app-c-subscribe__link govuk-link" %>
+  <%= link_to label, url, title: title, class: "app-c-subscribe__link govuk-link", data: data %>
```

Comment:
This `data` parameter is making the tests fail for the  clocks change page which [also uses this partial, apparently](https://github.com/alphagov/frontend/blob/85ddb3ef9ae62b359459861f61fdfe68df0c46f7/app/views/calendar/when_do_the_clocks_change.html.erb#L35).